alert.css

```diff
@@ -0,0 +1,50 @@
+.alert {
+    color: rgb(53, 52, 52);
+    font-family: "FontAwesome";
+    padding: 5px 10px;
+    /* box-shadow: 5px 10px; */
+    margin: 5px;
+}
+
+.alert .alert-title {
+    display: inline;
+}
+
+/* ------------------------------- specific alerts ----------------------- */
+
+.alert-success {
+    background-color: rgb(133, 214, 133);
+}
+
+.alert-error {
+    background-color: rgb(228, 88, 88);
+}
+
+.alert-warning {
+    background-color: rgb(236, 236, 82);
+}
+
+.alert-info {
+    background-color: rgb(92, 235, 235);
```

Comment:
Use css variables here, will definitely save your time a lot

Reply:
I have done the suggested changes